src/app/loadout-builder/loadout-builder-reducer.ts

```diff
@@ -521,7 +523,14 @@ function lbConfigReducer(defs: D2ManifestDefinitions) {
         return updateLoadout(state, setLoadoutParameters({ exoticArmorHash: lockedExoticHash }));
       }
       case 'removeLockedExotic':
-        return updateLoadout(state, setLoadoutParameters({ exoticArmorHash: undefined }));
+        return updateLoadout(
+          state,
+          setLoadoutParameters({ exoticArmorHash: undefined, perks: [] }),
```

Comment:
This would somehow need to be aware of which perks to remove when an exo class item is removed, instead of just wiping them entirely.